project.clj

```diff
@@ -3,6 +3,16 @@
   :url "https://github.com/weavejester/meta-merge"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]]
-  :aliases {"test-all" ["with-profile" "default:+1.6" "test"]}
-  :profiles {:1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}})
+  :plugins [[lein-cljsbuild "1.0.3"]
+            [lein-doo "0.1.6"]]
+  :aliases {"test-cljs" ["doo" "nashorn" "test" "once"]
+            "test-all" ["do" ["test"] ["test-cljs"]]}
+  :profiles {:provided
+             {:dependencies [[org.clojure/clojure "1.7.0"]
```

Comment:
You don't need to provide Clojure 1.7.0, since that'll be necessary no matter whether the end user is using Clojure or ClojureScript.

I also prefer to use `:scope` rather than the `:provided` profile, since that means all the main dependencies are in one place.

```clojure
[[org.clojure/clojure "1.7.0"]
 [org.clojure/clojurescript "1.7.189" :scope "provided"]]
```